src/doc/trpl/the-stack-and-the-heap.md

```diff
@@ -1,3 +1,526 @@
 % The Stack and the Heap
 
-Coming Soon
+As a systems language, Rust operates at a low level. If you’re coming from a
+high-level language, there’s some aspects of systems programming that you may
+not be familiar with. The most important one is how memory works, with a stack
+and a heap. If you’re familiar with the terms ‘stack’ and ‘heap’, this chapter
```

Comment:
Based on [this comment from @asolove](https://github.com/rust-lang/rust/pull/25292#issuecomment-100726362), you might consider refining this to:
> If you are familiar with the terms ‘stack’ and ‘heap’, and in particular with the distinction between a variable on a ‘stack frame’ versus a value allocated on the heap, this chapter ...

(both the terms "stack" and "heap" have meanings as abstract data-structures that have very little to do with what you are presenting here, and thus could mislead as to the content of this chapter.)